src/services/LoggingService.js

```diff
@@ -53,15 +53,14 @@ class LoggingService {
     } else if (error.request) {
       const { request } = error;
       const errorMessage = request.responseText || error.message;
-      const requestUrl = request.responseURL || (request.config && request.config.url);
       const requestAttributes = {
         errorType: 'api-request-error',
         errorStatus: request.status,
-        errorUrl: requestUrl,
+        errorUrl: request.responseURL,
         errorData: errorMessage,
       };
       updatedCustomAttributes = Object.assign({}, requestAttributes, customAttributes);
-      processedError = new Error(`API request failed: ${request.status} ${requestUrl} ${errorMessage}`);
+      processedError = new Error(`API request failed: ${request.status} ${request.responseURL} ${errorMessage} ${JSON.stringify(error)}`);
```

Comment:
@douglashall: You could add it as a custom metric using `fixErrorLength(JSON.stringify(error))` to get as much as possible.

Reply:
Does it matter?

Reply:
If it is temporary, it doesn't matter much.  You'll get a few more characters if it is big and you separate it out.